dataactbroker/fileRoutes.py

```diff
@@ -172,6 +174,17 @@ def check_current_page(submission):
 
         submission_id = submission.submission_id
 
+        # /v1/uploadDetachedFiles/
+        # DetachedFiles
+        detached_data = sess.query(Submission).filter(Submission.submission_id == submission_id,
```

Comment:
I'm not sure what the `Submission.d2_submission` does (is it working? Is it filtering as we expect? I don't know) but either way, this query isn't necessary. Because we have the entire submission in the `submission` variable we can' just check for `if submission.d2_submission` instead of `if detached_data.count() > 0:`, simplifying the check and removing an unnecessary DB call.